CompetitionTemplate/src/competition/subsystems/autonomous/selection/SetupRaiseArmAndTraverseCommand.java

```diff
@@ -22,18 +20,20 @@
     final DoubleProperty traverseMaxTime;
     final DoubleProperty initialAutoHeading;
     
+    final String label = "";
+    
     @Inject
     public SetupRaiseArmAndTraverseCommand(XPropertyManager propMan, 
             RaiseArmAndTraverseDefenseCommandGroup auto,
             AutonomousModeSelector autonomousModeSelector) {
         super(autonomousModeSelector);
         
-        traverseDefenseHeading = propMan.createPersistentProperty("traverseDefenseHeading", 90.0);
-        traverseDefensePower = propMan.createPersistentProperty("traverseDefensePower", 0.75);
-        autoArmGoal = propMan.createPersistentProperty("autonomousArmGoal", 30.0);
-        traverseMinTime = propMan.createPersistentProperty("traverseDefenseMinTime", 1.0);
-        traverseMaxTime = propMan.createPersistentProperty("traverseDefenseMaxTime", 3.0);
-        initialAutoHeading = propMan.createPersistentProperty("initialAutoHeading", PoseSubsystem.FACING_AWAY_FROM_DRIVERS);
+        traverseDefenseHeading = propMan.createPersistentProperty(label + "traverseDefenseHeading", 90.0);
+        traverseDefensePower = propMan.createPersistentProperty(label + "traverseDefensePower", 0.75);
+        autoArmGoal = propMan.createPersistentProperty(label + "autonomousArmGoal", 30.0);
+        traverseMinTime = propMan.createPersistentProperty(label + "traverseDefenseMinTime", 1.0);
+        traverseMaxTime = propMan.createPersistentProperty(label + "traverseDefenseMaxTime", 3.0);
+        initialAutoHeading = propMan.createPersistentProperty(label + "initialAutoHeading", PoseSubsystem.FACING_AWAY_FROM_DRIVERS);
```

Comment:
@JohnGilb I was wondering if there was a reason we wouldn't want this to always be the traverseDefenseHeading? Do we really need another property?